.github/workflows/test-desktop.yml

```diff
@@ -122,7 +122,7 @@ jobs:
           aws-access-key: ${{ secrets.AWS_S3_CACHE_ACCESS_KEY }}
           aws-secret-key: ${{ secrets.AWS_S3_CACHE_SECRET_KEY }}
       - name: Run unit tests
-        run: pnpm desktop test:jest
+        run: pnpm desktop test:jest:coverage
```

Comment:
Consider moving the unit tests to test.yml in order to collect coverage in one workflow.

Reply:
`test.yml` only includes tests for our libraries, am I wrong? If it's the case I think that we might want to keep them separated.